src/web/lib/components/AppHeader/index.js

```diff
@@ -19,6 +19,8 @@ export const AppHeader = ({ hasExtension, appVersion = null }) => {
         <header>
           <h1>Themer</h1>
           <h3>
+            A
+            {" "}
```

Comment:
What's this?

Reply:
Oof, yeah, prettier did that as part of breaking that line up but ensuring the space between "A" and "Firefox" is preserved. Kind of awkward.

Reply:
🤷‍♀️